test/test_format_az.py

```diff
@@ -0,0 +1,567 @@
+                         "altı yüz otuz bir min, səkkiz yüz doxsan üç")
+        self.assertEqual(pronounce_number(95505896639631893,
+                                          short_scale=False),
+                         "doxsan beş min beş yüz beş milyard, "
+                         "səkkiz yüz doxsan altı min altı yüz "
+                         "otuz doqquz milyon, altı yüz otuz bir min, "
+                         "səkkiz yüz doxsan üç")
+        self.assertEqual(pronounce_number(10e32, places=1),
+                         "bir dekilyon")
+
+        # infinity
+        self.assertEqual(
+            pronounce_number(sys.float_info.max * 2), "sonsuzluq")
+        self.assertEqual(
+            pronounce_number(float("inf")),
+            "sonsuzluq")
+        self.assertEqual(
+            pronounce_number(float("-inf")),
+            "mənfi sonsuzluq")
+
+    def test_ordinals(self):
+        self.assertEqual(pronounce_number(1, ordinals=True), "birinci")
+        self.assertEqual(pronounce_number(10, ordinals=True), "onuncu")
+        self.assertEqual(pronounce_number(15, ordinals=True), "on beşinci")
+        self.assertEqual(pronounce_number(20, ordinals=True), "iyirminci")
+        self.assertEqual(pronounce_number(27, ordinals=True), "iyirmi yeddinci")
+        self.assertEqual(pronounce_number(30, ordinals=True), "otuzuncu")
+        self.assertEqual(pronounce_number(33, ordinals=True), "otuz üçüncü")
+        self.assertEqual(pronounce_number(100, ordinals=True), "yüzüncü")
+        self.assertEqual(pronounce_number(1000, ordinals=True), "mininci")
+        self.assertEqual(pronounce_number(10000, ordinals=True),
+                         "on mininci")
+        self.assertEqual(pronounce_number(18691, ordinals=True),
+                         "on səkkiz min, altı yüz doxsan birinci")
+        self.assertEqual(pronounce_number(1567, ordinals=True),
+                         "min, beş yüz altmış yeddinci")
+        self.assertEqual(pronounce_number(1.672e-27, places=3,
+                                          scientific=True, ordinals=True),
+                         "bir nöqtə altı yeddi iki vurulsun on üstü mənfi "
+                         "iyirmi yeddinci")
+        self.assertEqual(pronounce_number(18e6, ordinals=True),
+                         "on səkkiz milyonuncu")
+        self.assertEqual(pronounce_number(18e12, ordinals=True,
+                                          short_scale=False),
+                         "on səkkiz milyardıncı")
+        self.assertEqual(pronounce_number(18e12, ordinals=True),
+                         "on səkkiz trilyonuncu")
+        self.assertEqual(pronounce_number(18e18, ordinals=True,
+                                          short_scale=False), "on səkkiz "
+                                                              "trilyonuncu")
+
+# def nice_time(dt, lang="en-us", speech=True, use_24hour=False,
+#              use_ampm=False):
```

Comment:
It looks like this has been copied across multiple languages :laughing: 

I'll remove it from all.